inference-engine/src/mkldnn_plugin/nodes/mkldnn_math_node.cpp

```diff
@@ -16,12 +16,15 @@ using namespace InferenceEngine;
 
 bool MKLDNNMathNode::isSupportedOperation(const std::shared_ptr<ngraph::Node>& op, std::string& errorMessage) noexcept {
     try {
-        if (initializers.find(op->get_type_info()) == initializers.end()) {
+        if (std::find_if(initializers.begin(), initializers.end(),
```

Comment:
This piece of code is used in a lot of places. Is it possible to have some helper function like `find_type_info(initializers, op->get_type_info())`?

Reply:
fixed